assets/js/modules/thank-with-google/components/setup/SetupMain.js

```diff
@@ -32,14 +32,14 @@ import { _x } from '@wordpress/i18n';
 import Data from 'googlesitekit-data';
 import SubscribeWithGoogleIcon from '../../../../../svg/graphics/logo-g.svg';
```

Comment:
@kuasha420 As per the AC, this should be updated to use the new TwG icon. Can you also rename the import from `SubscribeWithGoogleIcon` to `ThankWithGoogleIcon`